src/app/static/src/app/store/translations/locales.ts

```diff
@@ -572,6 +575,7 @@ const en: Translations = {
     uploadingStatus: "Uploading {{ fileNumber }} of {{ totalFiles }} (this may take a while)",
     uploadInputs: "Upload inputs",
     uploadComplete: "Upload complete",
+    uploadFileOutputComparison: "Comparison report",
```

Comment:
I suspect these might change to something wordier, pending what UNAIDS say about the mockups, but good to have a placeholder.